native-hello-module/README.md

```diff
@@ -1,54 +1,35 @@
-# GraalVM Demos: Building a HelloWorld Java Module into a Native Executable
-This repository contains the code for a demo application for [GraalVM](http://graalvm.org).
+# Build a Java Module into a Native Executable
 
-## Prerequisites
-* [GraalVM](http://graalvm.org)
-* [Native Image](https://www.graalvm.org/docs/reference-manual/native-image/)
-* Maven
+GraalVM Native Image supports the Java Platform Module System, introduced in Java 9, which means you can convert a modularized Java application into a native executable.
 
-## Preparation
+The `native-image` tool accepts the module-related arguments like `--module (-m)`, `--module-path (-p)`, `--add-opens`, `--add-exports` (same as for the java launcher). When such a module-related argument is used, the native-image tool itself is used as a module too.
 
-1. [Download GraalVM](https://www.graalvm.org/downloads/), unzip the archive, export the GraalVM home directory as the `$GRAALVM_HOME` and add `$GRAALVM_HOME/bin` to the `PATH` environment variable.
+In addition to supporting `--add-reads` and `--add-modules`, all module related options are considered prior to scanning the modulepath. This helps prevent class loading errors and allow for better module introspection at run time.
 
-  On Linux:
-  ```bash
-  export GRAALVM_HOME=/home/${current_user}/path/to/graalvm
-  export PATH=$GRAALVM_HOME/bin:$PATH
-  ```
-  On macOS:
-  ```bash
-  export GRAALVM_HOME=/Users/${current_user}/path/to/graalvm/Contents/Home
-  export PATH=$GRAALVM_HOME/bin:$PATH
-  ```
-  On Windows:
-  ```bash
-  setx /M GRAALVM_HOME "C:\Progra~1\Java\<graalvm>"
-  setx /M PATH "C:\Progra~1\Java\<graalvm>\bin;%PATH%"
-  ```
+The command to build a native executable from a Java module is:
+```bush
+native-image [options] --module <module>[/<mainclass>] [options]
+```
+The example below shows how to build a modular Java application into a native executable. 
 
-2. Install [Native Image](https://www.graalvm.org/docs/reference-manual/native-image/#install-native-image) by running:
-  ```bash
-  gu install native-image
-  ```
-For compilation, the `native-image` tool depends on the local toolchain.
-Please make sure that `glibc-devel`, `zlib-devel` (header files for the C library and zlib), and `gcc` are available on your system. Some Linux distributions may additionally require `libstdc++-static`.
-See [Prerequisites for Native Image](https://www.graalvm.org/dev/reference-manual/native-image/#prerequisites).
+## Preparation
 
-3. Download or clone the repository and navigate into the `native-hello-module` directory:
-  ```bash
-  git clone https://github.com/graalvm/graalvm-demos
-  cd graalvm-demos/native-hello-module
-  ```
-## Build the Application
-Before running this example, you need to build the application:
+1. Download and install the latest GraalVM JDK with Native Image using [GraalVM JDK Dowloader](https://github.com/graalvm/graalvm-jdk-downloader):
```

Comment:
using using [GraalVM JDK Dowloader] > using **the** [GraalVM JDK **Downloader**]
(see the spelling mistake)